dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/DefaultQueryParser.java

```diff
@@ -176,6 +196,10 @@ public Restriction getRestriction( Schema schema, String path, String operator,
             {
                 return Restrictions.notIlike( path, QueryUtils.parseValue( property.getKlass(), arg ), MatchMode.START );
             }
+            case "token":
```

Comment:
Would it make sense to add a `!token` operator also? for not matching, just create a new operator and ! the result..